src/main/kotlin/io/github/moulberry/notenoughupdates/util/KotlinRenderUtils.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2024 NotEnoughUpdates contributors
+ *
+ * This file is part of NotEnoughUpdates.
+ *
+ * NotEnoughUpdates is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * NotEnoughUpdates is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with NotEnoughUpdates. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.github.moulberry.notenoughupdates.util
+
+import net.minecraft.client.renderer.GlStateManager
+import net.minecraft.item.ItemStack
+import net.minecraft.util.ResourceLocation
+import org.lwjgl.opengl.GL11
+
+object KotlinRenderUtils {
+
+    @JvmStatic
+    fun renderItemStackButton(button: Rectangle, item: ItemStack, texture: ResourceLocation) {
```

Comment:
I feel like taking in a `Rectangle` here is a bit misleading, since the actual dimensions of the rectangle are entirely ignored (not used for the texture or centering the itemstack.